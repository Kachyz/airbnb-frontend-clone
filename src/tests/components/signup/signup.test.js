import React from 'react'
import { shallow } from 'enzyme'
import SignupForm from "../../../components/signup/form";

test('should render SignupForm page correctly', () => {
  const wrapper = shallow(<SignupForm />)

  wrapper.find('TextField').at(0).simulate('change',{
    target: {
      value: 'xxx',
      id: 'email'
    }
  })

  expect(wrapper.state('email')).toEqual("xxx")
  expect(wrapper).toMatchSnapshot()
})