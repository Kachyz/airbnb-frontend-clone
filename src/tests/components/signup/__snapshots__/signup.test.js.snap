// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignupForm page correctly 1`] = `
<WithStyles(Card)>
  Formulario de registro
  <TextField
    id="email"
    onChange={[Function]}
    placeholder="Email"
    required={false}
    select={false}
    value="xxx"
  />
  <TextField
    id="name"
    onChange={[Function]}
    placeholder="Name"
    required={false}
    select={false}
    value=""
  />
  <TextField
    id="lastName"
    onChange={[Function]}
    placeholder="LastName"
    required={false}
    select={false}
    value=""
  />
  <TextField
    id="password"
    onChange={[Function]}
    placeholder="Password"
    required={false}
    select={false}
    value=""
  />
  <TextField
    id="date"
    onChange={[Function]}
    required={false}
    select={false}
    type="date"
    value=""
  />
  <WithStyles(Button)
    color="secondary"
    onClick={[Function]}
    variant="contained"
  >
    Registrarse
  </WithStyles(Button)>
</WithStyles(Card)>
`;
